.github/workflows/native_jni_s3_pytorch.yml

```diff
@@ -84,32 +86,70 @@ jobs:
           aws s3 sync pytorch/pytorch-native/jnilib s3://djl-ai/publish/pytorch-${PYTORCH_VERSION}/jnilib
           aws cloudfront create-invalidation --distribution-id E371VB8JQ6NRVY --paths "/pytorch-${PYTORCH_VERSION}/jnilib*"
 
-  build-pytorch-jni-precxx11:
```

Comment:
Why do you remove precxx11? Although you switched the build env into CentOS7, the binary built (non-precxx11) cannot runs on any system with glibc 2.17.